packages/utils/test/baggage.test.ts

```diff
@@ -0,0 +1,92 @@
+import { createBaggage, getBaggageValue, parseBaggageString, serializeBaggage, setBaggageValue } from '../src/baggage';
+
+describe('Baggage', () => {
+  describe('createBaggage', () => {
+    it.each([
+      ['creates an empty baggage instance', {}, [{}, '']],
+      [
+        'creates a baggage instance with initial values',
+        { environment: 'production', anyKey: 'anyValue' },
+        [{ environment: 'production', anyKey: 'anyValue' }, ''],
+      ],
+    ])('%s', (_: string, input, output) => {
```

Comment:
This is very high up on the cleverness scale 😄 What do you think about splitting this up into separate tests for readability?

Reply:
I'm a big fan of table driven tests, and I believe they lend themselves well to maintainable and extensible unit tests. Picked it up from the golang ecosystem: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests.

It removes the boilerplate, and makes it easy to understand the that the setup + usage of the function is the exact same over the various scenarios. If the setup ever needs to change, just create a new test block!